src/elm/Explorer/View.elm

```diff
@@ -922,6 +931,64 @@ rejectionToItem ctx reason =
             , details = Nothing
             }
 
+        MissingBakerAddParameters ->
+            { content = [ text "A configure baker transaction is missing one or more arguments in order to add a baker" ]
+            , details = Nothing
+            }
+
+        UnexpectedBakerRemoveParameters ->
+            { content = [ text "A configure baker transaction to remove baker is passed unexpected arguments" ]
+            , details = Nothing
+            }
+            
+        CommissionsNotInRangeForBaking ->
+            { content = [ text "Not all baker commissions are within allowed ranges" ]
+            , details = Nothing
+            }
+            
+        AlreadyADelegator ->
+            { content = [ text "Tried to add baker for an account that already has a delegator" ]
+            , details = Nothing
+            }
+            
+        InsufficientBalanceForDelegationStake ->
+            { content = [ text "The amount on the account was insufficient to cover the proposed stake" ]
+            , details = Nothing
+            }
+
+        MissingDelegationAddParameters ->
+            { content = [ text "A configure delegation transaction is missing one or more arguments in order to add a delegator" ]
+            , details = Nothing
+            }
+
+        UnexpectedDelegationRemoveParameters ->
+            { content = [ text "A configure delegation transaction to remove delegation is passed unexpected arguments" ]
+            , details = Nothing
+            }
+
+        DelegatorInCooldown ->
+            { content = [ text "The change could not be made because the delegator is in cooldown" ]
+            , details = Nothing
+            }
+        NotADelegator addr ->
+            { content = [ text "Account ", viewAddress ctx <| T.AddressAccount addr, text " is not a delegation account" ]
+            , details = Nothing
+            }
+
+        DelegationTargetNotABaker targetId ->
+            { content = [ text "Delegation target with ID ", text <| String.fromInt targetId, text " is not a baker" ]
+            , details = Nothing
+            }
+
+        StakeOverMaximumThresholdForPool ->
+            { content = [ text "The amount would result in pool capital higher than the maximum threshold" ]
```

Comment:
Are we still using the word `pool`?
I am not exactly sure about what the final decision was regarding the new terminology.
But please make sure that this uses the correct terminology. Preferably also in variable/function names to avoid confusion.